css/css-grid-1/alignment/grid-content-alignment-must-ignore-collapsed-tracks-001.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Nit: I'd rename this to something simpler: `grid-content-alignment-collapsed-tracks-001` or `grid-content-distribution-collapsed-tracks-001`.